cmd/namespace/delete.go

```diff
@@ -38,9 +38,12 @@ func Delete(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "delete <name>",
 		Short: "Delete a namespace",
-		Args:  utils.ExactArgsAccepted(1, ""),
+		Args:  utils.MaximumNArgsAccepted(1, ""),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			nsToDelete := args[0]
+			nsToDelete := okteto.Context().Namespace
```

Comment:
I think we should move this block below the `NewContextCommand` so that we don't panic here if the context is not set correctly

Reply:
Fixed!